src/module/quickDiff.js

```diff
@@ -16,7 +17,7 @@ var quickDiff = function (param) {
   if ($('[href*="mediawiki.diff.styles"]').length < 1) {
     mw.loader.load(
       mw.util.wikiScript('load') +
-        '?modules=mediawiki.legacy.shared|mediawiki.diff.styles&only=styles',
+      '?modules=mediawiki.legacy.shared|mediawiki.diff.styles&only=styles',
```

Comment:
Do not modify the indents, please. Run `yarn pretty` if you are not sure what to do :)